src/_locales/de/messages.json

```diff
@@ -36,15 +36,15 @@
         "description": "Shown in the popup on disabled sites."
     },
     "popup_disable_for_site": {
-        "message": "Für diese Site deaktivieren",
+        "message": "Für diese Webseite deaktivieren",
```

Comment:
The special non-breaking blank spaces are here on purpose and should be preserved.

Reply:
Used Seite again and put back the non-breaking blank spaces